storio-content-resolver-annotations-processor-test/src/test/resources/CreatorWithNotMatchingArguments.java

```diff
@@ -0,0 +1,17 @@
+package com.pushtorefresh.storio.contentresolver.annotations;
+
+@StorIOContentResolverType(uri = "content://uri")
+public class CreatorWithNotMatchingArguments {
+
+    private long id;
+
+    @StorIOContentResolverColumn(name = "id", key = true)
+    long id() {
+        return id;
+    }
+
+    @StorIOContentResolverCreator
+    public CreatorWithNotMatchingArguments(long it) {
```

Comment:
Maybe rename to "field" and "anotherField" or smth else.
Example in misprint style is fun but as for me not so notable=(